ascii_symbols.h

```diff
@@ -0,0 +1,5 @@
+const int NEW_LINE = 10;
+const int ESC = 27;
+const int CANCEL = 24;
+const int BACKSPACE = 127;
+const char QUIT[4] = "quit";
```

Comment:
Sorry, I wasn't clear enough. What if user enter not "quit", but "Quit", or "quiT" for that matter?

Reply:
It won't work. Only "quit"

Reply:
It has to. Please use [tolower()|https://www.tutorialspoint.com/c_standard_library/c_function_tolower.htm] during comparison of the character.

Reply:
@ikakok Done